internal/resources/cluster/tkgvsphere/resource_tkgvsphere.go

```diff
@@ -576,12 +576,14 @@ var tkgVsphereNodePoolSpec = &schema.Schema{
 				Description: "Node labels",
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
+				Deprecated:  "node labels is not supported in TKGm vSphere",
 			},
 			cloudLabelKey: {
 				Type:        schema.TypeMap,
 				Description: "Cloud labels",
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
+				Deprecated:  "cloud labels is not supported in TKGm vSphere",
```

Comment:
deprecated is good, but 
is the main problem of the diff being reported addressed ?

Reply:
Seems better to remove the field as backend is not supported